zebra-chain/src/amount.rs

```diff
@@ -538,6 +538,10 @@ impl Constraint for NegativeAllowed {
 /// );
 /// ```
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Default)]
+#[cfg_attr(
+    any(test, feature = "proptest-impl"),
+    derive(proptest_derive::Arbitrary)
+)]
 pub struct NonNegative;
```

Comment:
Is there a way around this? There's an `Arbitrary` impl for `Amount<C: Constraint + Debug>`, it seems like it shouldn't be necessary to derive this explicitly here.